### ISSUE TITLE
jess/firefox needs --ipc=host

### ISSUE THREAD

**Issue by mviereck** [jessfraz/dockerfiles#360]
Previously `jess/firefox` could be started with e.g.
```
xhost +     # yeah I know, security issue
docker run -it \
    -v /tmp/.X11-unix:/tmp/.X11-unix \
    -e DISPLAY=unix$DISPLAY \
    jess/firefox
```
This error occurs:
```
(firefox:1): Gdk-ERROR **: The program 'firefox' received an X Window System error.
This probably reflects a bug in the program.
The error was 'BadValue (integer parameter out of range for operation)'.
  (Details: serial 335 error_code 2 request_code 130 (unknown) minor_code 3)
  (Note to programmers: normally, X errors are reported asynchronously;
   that is, you will receive the error a while after causing it.
   To debug your program, run it with the GDK_SYNCHRONIZE environment
   variable to change this behavior. You can then get a meaningful
   backtrace from your debugger if you break on the gdk_x_error() function.)
ExceptionHandler::GenerateDump cloned child 89
ExceptionHandler::SendContinueSignalToChild sent continue signal to child
ExceptionHandler::WaitForContinueSignal waiting for continue signal...
```
That can be fixed with disabling IPC namespacing. Though, that degrades container isolation:
```
--ipc=host
```
Most probably firefox tries to access shared memory from host and fails. I often get a similar issue due to shared memory of X server (extension `MIT-SHM`). For other applications than firefox it is enough to disable extension `MIT-SHM` in additional X servers. I'll discuss that in ticket #359.

**Comment by mviereck** [comment]
It seems the issue is fixed in #376 with setting `browser.tabs.remote.autostart` to `false` in `about:config`. Though, the fix in #376 only takes effect if running firefox as root in container.

**Comment by jessfraz** [comment]
fixed by https://github.com/jessfraz/dockerfiles/pull/429 thanks!
